package.json

```diff
@@ -15,6 +15,7 @@
     "node-fetch": "^2.3.0",
     "passport": "^0.4.0",
     "passport-discord": "^0.1.3",
+    "stopword": "^0.2.1",
```

Comment:
this package is not used anywhere?

Reply:
it is need for the related resource module to work properly.
![stopword](https://user-images.githubusercontent.com/37092867/54829707-490fa900-4cb7-11e9-97c0-31566d6f5edd.png)
![stopword](https://user-images.githubusercontent.com/37092867/54829726-52007a80-4cb7-11e9-9d25-ab0b1a77729e.png)